mantidproject/mantid
# Quasielastic fit functions with conv fit

Fixes #12669 

The ConvFit tab in indirect data analysis now allows for QuasiElastic functions to be used.
The code has also been changed from the previous hard coded approach to have a far better coverage for generalised code.
# To test:
- Open the Convfit tab (from the menu select: Interfaces>Indirect>Data Analysis then click the ConvFit tab)
- Run any of the newly added QuasiElastic functions (selected in the FitType dropdown menu)
  - Load sample relevant data
  - Adjust parameters via the property tree
  - Click run
- Check the output is as expected 

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24523&oldid=24462).
